docs/develop/03_workshop/03_overview.md

```diff
@@ -53,3 +74,23 @@ participant B as KILT Blockchain
     C-->>-V: submit presentation
     V->>B: check validity of presentation
 ```
+
+### Example: Requesting a travel visa
+
+To take an example of applying for a travel visa:
+
+1. The Embassy (analogous to the Verifier) asks a traveler (analogous to the Claimer) for a specific document or CType. For example, it could be a bank statement. The Embassy asks, "Provide proof of financial stability, and we'll grant you a visa." The traveler gets the bank statement from their bank, gets it attested by the bank (The Attester), and prepares the document.
+2. The document is ready, but the Embassy doesn't need all the information in the document. The embassy wants to know if a traveler has sufficient funds, but they don't need to know any transaction details. The traveler redacts or hides these details while presenting.
+3. The traveler presents the document to the embassy.
+4. The embassy verified the document's authenticity by comparing its hash value with the one on their internal system or a decentralized ledger.
+5. Since they trust the Attester (in this case, the bank that attested the bank statement), they approved the visa application.
+
+:::tip Summary
+
+As you can see, the Embassy didn't need to trust the Claimer directly in this system.
+They trust the Attester, whom they had previously worked with, or respect due to their position.
+And with that trust, they grant the visa with no knowledge of what the Claimer has used the credential for.
```

Comment:
@Dudleyneedham 

> https://github.com/KILTprotocol/docs/pull/284#discussion_r1384859800

Does this represent what you meant?

Reply:
Yes :)